src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -40,7 +44,31 @@
      * София - 900</p>
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException {
+        if (stringToDate(dateFrom).compareTo(stringToDate(dateTo)) > 0) {
```

Comment:
there are more relevant method than `compareTo`. don't use it at all

Reply:
Done